README.md

```diff
@@ -128,23 +134,25 @@ export class AppComponent {
 
 Available options based on lightbox2 options
 
-Properties | Default | Description
------------|---------|------------
-fadeDuration | **0.7** seconds | *duration* starting when the **src** image is **loaded** to **fully appear** onto screen.
-resizeDuration | **0.5** seconds | *duration* starting when Lightbox container  **change** its dimension from a *default/previous image* to the *current image* when the *current image* is **loaded**.
-fitImageInViewPort | **true** | Determine whether lightbox will use the natural image *width/height*  or change the image *width/height* to fit the view of current window. Change this option to **true** to prevent problem when image too big compare to browser windows.
-positionFromTop | **20** px | The position of lightbox from the top of window browser
-showImageNumberLabel | **false** | Determine whether to show the image number to user. The default text shown is `Image IMAGE_NUMBER of ALBUM_LENGTH`
-alwaysShowNavOnTouchDevices | **false** | Determine whether to show `left/right` arrow to user on Touch devices.
-wrapAround | **false** | Determine whether to move to the start of the album when user reaches the end of album and vice versa. Set it to **true** to enable this feature.
-disableKeyboardNav | **false** | Determine whether to disable navigation using keyboard event.
-disableScrolling | **false** | If **true**, prevent the page from scrolling while Lightbox is open. This works by settings overflow hidden on the body.
-centerVertically | **false** | If **true**, images will be centered vertically to the screen.
-albumLabel | 	"Image %1 of %2" | 	The text displayed below the caption when viewing an image set. The default text shows the current image number and the total number of images in the set.
-enableTransition | **true** | Transition animation between images will be disabled if this flag set to **false**
+| Properties                  | Default          | Description                                                                                                                                                                                                                                 |
```

Comment:
any ideas why these files got change ?

Reply:
Guessing this was some IDE format thingy.